docs/deploy-coe-with-Kafka.md

```diff
@@ -299,7 +299,7 @@ The following example shows the YAML file with the data format configured as JSO
             "Endpoints": {
               "KAFKA": {
                 "DataFormat": "JSON",
-                "ServerUrl": "X.X.X.X:9092", #Specify the Kafka broker IP
+                "ServerUrl": "X.X.X.X:9092,Y.Y.Y.Y:9092", #Specify the Kafka broker IP
```

Comment:
```suggestion
                "ServerUrl": "X.X.X.X:9092,Y.Y.Y.Y:9092", #Specify the bootstrap Kafka broker addresses (separated by comma in case multiple bootstrap brokers are to be configured)
```